doc/en/user/source/configuration/virtual-services.rst

```diff
@@ -133,6 +172,8 @@ To disable global services, log into the GeoServer web administration interface
 Isolated Workspaces
 ^^^^^^^^^^^^^^^^^^^
 
+When publishing XML content each FeatureType is expected to be published into a unique XML Namespace. Isolated workspaces allow GeoServer to configure two workspaces with the same XML Namespace with some restrictions outlined below to ensure there is no opportunity for namespace conflict.
```

Comment:
this is confusing - perhaps say something along the lines that, normally, a workspace equals a xml namespace.  However, there is a special way to have workspaces with features in the same XML namespace.

Reply:
I had a go at using more words. It is a case where it is obvious to those familiar with XML, and annoying for everyone else.